.github/workflows/release.yml

```diff
@@ -42,25 +42,9 @@ jobs:
           go-version-file: "go.mod"
           cache: ${{ runner.arch != 'arm64' }}
 
-      - if: matrix.os == 'windows-latest'
-        name: Install dependencies in windows
-        uses: crazy-max/ghaction-chocolatey@v2
-        with:
-          args: install make wget zip
-
-      - name: setup env
-        run: |
-          echo "::set-env name=GOPATH::$(go env GOPATH)"
-          echo "::add-path::$(go env GOPATH)/bin"
-        shell: bash
-        env:
-          ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
```

Comment:
can you describe the effect f removing these two actions in the PR descriptions? i don't know why they are changed/removed

Reply:
added to PR description 🙂